frank.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * A script to frank the build i.e. to copy the package.json an readme files to the dist folder.
+ * This will enable importing icons from subfolders with cleaner paths. For example 'import {Alert} from suomifi-icons/base/'
```

Comment:
`import {Alert} from suomifi-icons/base/` you cannot import like this anymore, right? 

Isn't it `import { IconAlert } from 'suomifi-icons'`?

Reply:
Should be fixed now.